media-server/media_server.sh

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#
+# Author: Fabian Solano <fabian.solano@ridgerun.com>
+# 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+
+# Load settings
+source config.txt
+
+# Create pipelines
+echo "Creating camera pipeline"
+gstd-client pipeline_create cam_pipe v4l2src device=$CAMERA \
+! video/x-raw, width=$CAMERA_WIDTH, height=$CAMERA_HEIGHT, \
+framerate=10/1, format=$CAMERA_FORMAT ! videoconvert \
+! video/x-raw, format=Y42B \
+! interpipesink name=cam_pipe_src sync=false async=false
+
+echo "Creating input stream pipeline"
+gstd-client pipeline_create in_stream_pipe rtspsrc \
+location=$RTSP_URI ! rtph264depay ! h264parse ! avdec_h264 ! queue \
```

Comment:
Prefer decodebin.

Reply:
Actually if you use an `uridecodebin` you can receive any stream, not limited to RTSP.